### DISCUSSION TITLE
Will our private collaborators issues be removed when we migrate to Github enterprise cloud?

### DISCUSSION THREAD

**Discussion by AA-Hamza** [community/community#27236] [Enterprise]
Hello,
We are trying to migrate from a personal private repo (github teams) into an organization (github enterprise cloud) and we have many outside collaborators that we would like to keep attached to their issues & PRs. Now it is mentioned in the [docs (point 4)](https://docs.github.com/en/enterprise-cloud@latest/repositories/creating-and-managing-repositories/transferring-a-repository#whats-transferred-with-a-repository) that issues for people outside the organization will be cleared.

> When a repository is transferred between two personal accounts, issue assignments are left intact. When you transfer a repository from a personal account to an organization, issues assigned to members in the organization remain intact, and all other issue assignees are cleared. Only owners in the organization are allowed to create new issue assignments. When you transfer a repository from an organization to a personal account, only issues assigned to the repository's owner are kept, and all other issue assignees are removed.

We tried a little demo using github enterprise trial and transferred ownership of private repo to an organization and github added people outside the organization as outside collaborators and it kept their issues & PRs.

Is it safe to just transfer ownership without adding people to the organization?
Do we have to add outside collaborators as members or not?

**Comment by ssorallen** [comment]
@AA-Hamza Have you figured out how to transfer from github.com -> GitHub Enterprise Cloud? Our organization is not publicly visible so I'm not able to transfer repos to it. I'm not clear on how visibility of enterprises and organizations works for Enterprise Cloud.

**Reply by AA-Hamza** [reply]
We will make the move in a very short days and we still have so many questions, the docs are definitely lacking. My advice is you can make a replica of your situation using a dummy and a testing organization with github enterprise cloud trial (no credit card needed), this cleared some of our questions.
